assets/js/src/components/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
I made separate `component` and `container` directories, you can read more about the difference here https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0